File: src/containsDuplicate.java
import java.util.Arrays;

public class containsDuplicate {
    public static boolean containsDuplicate(int[] nums) {
        Arrays.sort(nums);
        for (int i = 0; i < nums.length - 1; i++){
            if (nums[i] == nums[i+1]) {
                return true;
            }
        }
        return false;
    }

        public static void main (String[]args){
            int[] values = {0,2,2,0};
            System.out.println(containsDuplicate(values));
    }
}

File: src/ContainsDuplicate.cpp
class Solution {
public:
    bool containsDuplicate(vector<int>& nums) {
        // sort values
        sort(nums.begin(), nums.end());
        
        for (int i = 0; i < nums.size() - 1; i++) {
            // check if curr == next
            if (nums[i] == nums[i+1]) return true;
        }
        return false;
    }
};
